src/components/Form/CreateAccount.tsx

```diff
@@ -72,6 +72,8 @@ interface AccountCreationFormProps {
 
 function AccountCreationForm(props: AccountCreationFormProps) {
   const { errors, formValues, setFormValue } = props
+  const [pendingConfirmation, setPendingConfirmation] = React.useState<React.SyntheticEvent | null>(null)
```

Comment:
Super subtle issue here: We cannot actually store the event here, since we will only use the event to do a `.preventDefault()` in the parent component.

But: `preventDefault()` only works when called synchronously.

Solution: Let's call `event.preventDefault()` in this component now that it sets up its own `onSubmit` handler anyway and not expose the event in its `props.onSubmit()`.